openverse_catalog/docs/adding_a_new_provider.md

```diff
@@ -0,0 +1,90 @@
+```
+
+You should see output similar to this:
+```
+Creating files in /Users/staci/projects/openverse-projects/openverse-catalog
+API script:        openverse-catalog/openverse_catalog/dags/providers/provider_api_scripts/foobar_museum.py
+API script test:   openverse-catalog/tests/dags/providers/provider_api_scripts/test_foobar_museum.py
+
+NOTE: You will also need to add a new ProviderWorkflow dataclass configuration to the PROVIDER_WORKFLOWS list in `openverse-catalog/dags/providers/provider_workflows.py`.
+```
+
+This generates a provider script with a templated `ProviderDataIngester` for you in the [`provider_api_scripts` folder](../dags/providers/provider_api_scripts), as well as a corresponding test file. Complete the TODOs detailed in the generated files to implement behavior specific to your API.
+
+Some APIs may not fit perfectly into the established `ProviderDataIngester` pattern. For advanced use cases and examples of how to modify the ingestion flow, see the [`ProviderDataIngester` FAQ](provider_data_ingester_faq.md).
+
+
+### Add a `ProviderWorkflow` configuration class
+
+Now that you have an ingester class, you're ready to wire up a provider DAG in Airflow to automatically pull data and load it into our Catalog database. This is as simple as defining a `ProviderWorkflow` configuration dataclass and adding it to the `PROVIDER_WORKFLOWS` list in [`provider_workflows.py`](../dags/providers/provider_workflows.py). Our DAG factories will pick up the configuration and generate a complete new DAG in Airflow!
+
+At minimum, you'll need to provide the following in your configuration:
+* `provider_script`: the name of the file where you defined your `ProviderDataIngester` class
+* `ingestion_callable`: the `ProviderDataIngester` class itself
+* `media_types`: the media types your provider handles
+
+Example:
+```python
+# In openverse_catalog/dags/providers/provider_workflows.py
+from providers.provider_api_scripts.foobar_museum import FoobarMuseumDataIngester
+
+...
+
+PROVIDER_WORKFLOWS = [
+    ...
+    ProviderWorkflow(
+        provider_script='foobar_museum',
+        ingestion_callable=FooBarMuseumDataIngester,
+        media_types=("image", "audio",)
+    )
+]
+```
+
+There are many other options that allow you to tweak the `schedule` (when and how often your DAG is run), timeouts for individual steps of the DAG, and more. These are documented in the definition of the `ProviderWorkflow` dataclass. *<TODO: add docs for other options.>*
+
+After adding your configuration, run `just up` and you should now have a fully functioning provider DAG! *<TODO: add and link to docs for how to run provider DAGs locally, preferably with images.>* *NOTE*: when your code is merged, the DAG will become available in production but will be disabled by default. A contributor with Airflow access will need to manually turn the DAG on in production.
```

Comment:
I would split this across lines, it seems like markdown doesn't render the `* *` very well:
```suggestion
After adding your configuration, run `just up` and you should now have a fully functioning provider DAG! *<TODO: add and link to docs for how to run provider DAGs locally, preferably with images.>*

**NOTE**: when your code is merged, the DAG will become available in production but will be disabled by default. A contributor with Airflow access will need to manually turn the DAG on in production.
```